[2022-04-27T20:05:26.234] [ERROR] error - Servidor escuchando en puerto 8080
[2022-04-27T20:05:32.390] [ERROR] error - Servidor escuchando en puerto 8080
[2022-04-27T20:24:54.484] [ERROR] error - ruta '/' metodo GET no implementado
[2022-04-27T20:37:09.823] [ERROR] error - Error: No recipients defined
    at SMTPConnection._formatError (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:784:19)
    at SMTPConnection._setEnvelope (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:1001:34)
    at SMTPConnection.send (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:615:14)
    at sendMessage (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-transport\index.js:227:28)
    at C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-transport\index.js:285:25
    at SMTPConnection._actionAUTHComplete (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:1550:9)
    at SMTPConnection.<anonymous> (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:540:26)
    at SMTPConnection._processResponse (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:947:20)
    at SMTPConnection._onData (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:749:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:189:44) {
  code: 'EENVELOPE',
  command: 'API'
}
[2022-04-27T20:37:10.723] [ERROR] error - RestException [Error]: Authenticate
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2022-04-27T20:38:06.095] [ERROR] error - RestException [Error]: Authenticate
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2022-04-27T20:42:32.546] [ERROR] error - RestException [Error]: Authenticate
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2022-04-27T20:43:37.321] [ERROR] error - RestException [Error]: Authenticate
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2022-04-27T20:44:08.629] [ERROR] error - RestException [Error]: Authenticate
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: undefined
}
[2022-04-27T20:54:29.361] [ERROR] error - RestException [Error]: The 'To' number 456 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-27T20:54:30.223] [ERROR] error - RestException [Error]: The From phone number +14155238886 is not a valid, SMS-capable inbound phone number or short code for your account.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21606,
  moreInfo: 'https://www.twilio.com/docs/errors/21606',
  details: undefined
}
[2022-04-27T20:56:19.802] [ERROR] error - RestException [Error]: The 'To' number 456 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-27T20:56:20.715] [ERROR] error - RestException [Error]: The number  is unverified. Trial accounts cannot send messages to unverified numbers; verify  at twilio.com/user/account/phone-numbers/verified, or purchase a Twilio number to send messages to unverified numbers.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21608,
  moreInfo: 'https://www.twilio.com/docs/errors/21608',
  details: undefined
}
[2022-04-27T20:57:47.548] [ERROR] error - RestException [Error]: The 'To' number 456 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-27T21:01:33.609] [ERROR] error - RestException [Error]: The 'To' number 456 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-27T22:48:23.646] [ERROR] error - ruta '/' metodo POST no implementado
[2022-04-27T22:48:32.143] [ERROR] error - ruta '/' metodo POST no implementado
[2022-04-27T22:48:37.801] [ERROR] error - ruta '/' metodo POST no implementado
[2022-04-27T22:48:46.190] [ERROR] error - ruta '/' metodo POST no implementado
[2022-04-27T22:50:24.024] [ERROR] error - ruta '/' metodo POST no implementado
[2022-04-27T22:59:17.954] [ERROR] error - Error: No recipients defined
    at SMTPConnection._formatError (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:784:19)
    at SMTPConnection._setEnvelope (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:1001:34)
    at SMTPConnection.send (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:615:14)
    at sendMessage (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-transport\index.js:227:28)
    at C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-transport\index.js:285:25
    at SMTPConnection._actionAUTHComplete (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:1550:9)
    at SMTPConnection.<anonymous> (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:540:26)
    at SMTPConnection._processResponse (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:947:20)
    at SMTPConnection._onData (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:749:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:189:44) {
  code: 'EENVELOPE',
  command: 'API'
}
[2022-04-27T22:59:29.492] [ERROR] error - Error: No recipients defined
    at SMTPConnection._formatError (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:784:19)
    at SMTPConnection._setEnvelope (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:1001:34)
    at SMTPConnection.send (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:615:14)
    at sendMessage (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-transport\index.js:227:28)
    at C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-transport\index.js:285:25
    at SMTPConnection._actionAUTHComplete (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:1550:9)
    at SMTPConnection.<anonymous> (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:540:26)
    at SMTPConnection._processResponse (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:947:20)
    at SMTPConnection._onData (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:749:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:189:44) {
  code: 'EENVELOPE',
  command: 'API'
}
[2022-04-27T22:59:50.339] [ERROR] error - Error: No recipients defined
    at SMTPConnection._formatError (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:784:19)
    at SMTPConnection._setEnvelope (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:1001:34)
    at SMTPConnection.send (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:615:14)
    at sendMessage (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-transport\index.js:227:28)
    at C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-transport\index.js:285:25
    at SMTPConnection._actionAUTHComplete (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:1550:9)
    at SMTPConnection.<anonymous> (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:540:26)
    at SMTPConnection._processResponse (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:947:20)
    at SMTPConnection._onData (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:749:14)
    at TLSSocket.SMTPConnection._onSocketData (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\nodemailer\lib\smtp-connection\index.js:189:44) {
  code: 'EENVELOPE',
  command: 'API'
}
[2022-04-27T23:06:33.334] [ERROR] error - RestException [Error]: The 'To' number 123 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-27T23:06:34.182] [ERROR] error - RestException [Error]: The 'To' number 123 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-27T23:21:58.931] [ERROR] error - RestException [Error]: The 'To' number 123 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-27T23:21:59.782] [ERROR] error - RestException [Error]: The 'To' number 123 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-27T23:22:44.173] [ERROR] error - RestException [Error]: The 'To' number 123 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-27T23:22:44.994] [ERROR] error - RestException [Error]: The 'To' number 123 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-28T15:53:04.032] [ERROR] error - ruta '/' metodo GET no implementado
[2022-04-28T16:47:57.953] [ERROR] error - RestException [Error]: The 'To' number 654 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-28T16:47:58.807] [ERROR] error - RestException [Error]: The 'To' number 654 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-28T19:25:11.630] [ERROR] error - ruta '/' metodo GET no implementado
[2022-04-28T19:25:27.982] [ERROR] error - ruta '/' metodo GET no implementado
[2022-04-28T19:27:36.174] [ERROR] error - Invalid password
[2022-04-28T19:51:55.429] [ERROR] error - El email ya se encuentra registrado
[2022-04-28T19:54:14.863] [ERROR] error - El email ya se encuentra registrado
[2022-04-28T19:54:45.399] [ERROR] error - El email ya se encuentra registrado
[2022-04-28T19:55:40.840] [ERROR] error - El email ya se encuentra registrado
[2022-04-29T08:09:12.875] [ERROR] error - ruta '/' metodo GET no implementado
[2022-04-29T08:23:06.818] [ERROR] error - Error: Authenticate
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: 'undefined'
}
[2022-04-29T08:23:07.572] [ERROR] error - Error: Authenticate
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 401,
  code: 20003,
  moreInfo: 'https://www.twilio.com/docs/errors/20003',
  details: 'undefined'
}
[2022-04-29T08:25:10.035] [ERROR] error - RestException [Error]: The 'To' number 456 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-04-29T08:25:10.870] [ERROR] error - RestException [Error]: The 'To' number 456 is not a valid phone number.
    at success (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\twilio\lib\base\Version.js:135:15)
    at Promise_then_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:766:44)
    at Promise_done_fulfilled (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:835:31)
    at Fulfilled_dispatch [as dispatch] (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1229:9)
    at Pending_become_eachMessage_task (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\q\q.js:1369:30)
    at RawTask.call (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\asap.js:40:19)
    at flush (C:\Users\Alan\Documents\desafio.clase.36.tercer.preentrega\node_modules\asap\raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:77:11) {
  status: 400,
  code: 21211,
  moreInfo: 'https://www.twilio.com/docs/errors/21211',
  details: undefined
}
[2022-05-02T19:21:27.661] [ERROR] error - El email ya se encuentra registrado
[2022-05-02T20:02:15.396] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T20:02:21.706] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T20:05:03.314] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T20:09:38.781] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T20:10:01.255] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T20:10:15.636] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T20:11:39.487] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T20:25:01.734] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T20:25:26.445] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T20:26:02.447] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:30:15.140] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:30:34.580] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:30:42.644] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:31:04.740] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:31:38.392] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:33:15.077] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:35:03.102] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:35:48.707] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:36:22.063] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:38:11.477] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:38:28.642] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:39:09.486] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:42:48.149] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:43:01.106] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:43:29.913] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:44:38.698] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:45:41.632] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:46:00.228] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:47:28.386] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:48:52.353] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-02T21:49:14.160] [ERROR] error - ruta '/' metodo GET no implementado
[2022-05-03T18:20:00.387] [ERROR] error - ruta '/' metodo GET no implementado
